app/logic/events.py

```diff
@@ -450,3 +451,11 @@ def getTomorrowsEvents():
     tomorrowDate = date.today() + timedelta(days=1)
     events = list(Event.select().where(Event.startDate==tomorrowDate))
     return events
+def eventViewCount(viewer,event):
+    """This checks if the current user already viewed the event. If not, insert a recored to EventView table"""
+    event_view_exists = EventView.select().where(EventView.user == viewer, EventView.event == event).exists()     
```

Comment:
We use camleCase for variable naming/declaration across our logic files, please stay consistent.